src/test/java/uk/gov/pay/api/it/validation/PaymentResourceSearchValidationITest.java

```diff
@@ -21,6 +21,8 @@
 public class PaymentResourceSearchValidationITest extends PaymentResourceITestBase {
 
     private static final String VALID_REFERENCE = "test_reference";
+    private static final String VALID_LAST_DIGITS_CARD_NUMBER = "4242";
+    private static final String VALID_FIRST_DIGITS_CARD_NUMBER = "424242";
```

Comment:
Minor but I’d feel slightly more confident if one of these strings weren’t a subset of the other…

Reply:
👍